pkg/buffers/pool/pool.go

```diff
@@ -25,25 +25,19 @@ func (poolMetrics) recordBufferReturn(buf *buffer.Buffer) {
 	buf.RecordMetric()
 }
 
-// Opts is a function that configures a BufferPool.
-type Opts func(pool *Pool)
-
 // Pool of buffers.
 type Pool struct {
 	*sync.Pool
-	bufferSize uint32
+	bufferSize int
```

Comment:
Why did you change this?

Reply:
The main reason is convenience. Using an `int` instead of a `uint32` eliminates the need to cast in all the places where it's used.